### PR TITLE
Introduce docker 

### PR DESCRIPTION
**Pull request by TanmayPatil105**
## What is this Pull Request About?
Dockerfile created to build and test `searchor`.

## What will this Pull Request Affect?

FIles `Dockerfile`, `.dockerignore` and `requirements.txt` have been added.

# Any additional information?

README.md can be modified and the following line can be added.
```Docker
sudo docker built -t searchor .
```

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pyperclip
```
Comment (ArjunSharda) [inline requirements.txt:1]:
```suggestion
pyperclip, aenum
```

**Review by ArjunSharda — CHANGES_REQUESTED** [review]
Hi! I think the custom uses aenum, so i've requested a change for that.

**Comment by TanmayPatil105** [comment]
I think we do not need requirements.txt. They are getting installed during installation of searchor since they are mentioned in `setup.py`.

## Screenshot 
image

**Review by ArjunSharda — APPROVED** [review]
Thanks for the PR, LGTM! Happy Hacktoberfest!

**Review by itskegnh — APPROVED** [review]

**Comment by TanmayPatil105** [comment]
Thanks for merging :)

**Pull request merged**